### ISSUE TITLE
remove `import distutils.sysconfig` from `lib/matplotlib/__init__.py`

### ISSUE THREAD

**Issue by sandrotosi** [matplotlib/matplotlib#11021]
Hello,
due to a recent change in how python3 packages are handled, we found out that `lib/matplotlib/__init__.py` still contains an unused `import distutils.sysconfig` - could you remove it?

thanks!

**Comment by tacaswell** [comment]
Sure, but what is the consequence of the extra import?

**Comment by sandrotosi** [comment]
(sorry i thought i explained but i edited a couple of times the post and it got lost) in debian the whole distutils package used to be provided by the interpreter packages; now it has been split out so all the packages that previously were able to use `distutils` implicitly now have to add an explicit dependency on that package. since mpl doesnt actually need `distutils.sysconfig` at this point, the right approach seems to be to remove the import altogether.

**Comment by tacaswell** [comment]
We do use it in `setupext.py` so it needs to stay as at least a build dependency.

**Comment by sandrotosi** [comment]
yes, you're right: it's used in setupext.py but (at least for debian) we have a different set of dependencies for when to build the package (and distutils is there) and when to install it for users to use it.
